docs/en/operations/settings/settings.md

```diff
@@ -516,11 +531,26 @@ This method might seem primitive, but it doesn't require external data about net
 Thus, if there are equivalent replicas, the closest one by name is preferred.
 We can also assume that when sending a query to the same server, in the absence of failures, a distributed query will also go to the same servers. So even if different data is placed on the replicas, the query will return mostly the same results.
 
-### in_order
+### In Order {#load_balancing-in_order}
+
+```
+load_balancing = in_order
+```
 
 Replicas are accessed in the same order as they are specified. The number of errors does not matter.
```

Comment:
The number of errors does matter here irc. @alesapin, maybe you are good person to check and confirm that.

Reply:
Looks true.